app/src/pages/inside/projectSettingsPageContainer/projectSettingsPageContainer.jsx

```diff
@@ -166,10 +167,17 @@ export const ProjectSettingsPageContainer = () => {
     return itemsConfig[activeTab].component;
   }, [activeTab, createItemsConfig, dispatch]);
 
+  const header = useMemo(() => {
```

Comment:
Is it necessary to memo such plain component?
You don't drill down it as a prop and it doesn't contain any calculations.